elements/pfe-card/src/pfe-card.scss

```diff
@@ -1,127 +1,149 @@
-  --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--lightest--link--focus)};
-}
-
-:host([color="complement"]) {
-  --pfe-card--bg:                               #{pfe-color(surface--complement)};
-  --pfe-broadcasted--color--text:               #{pfe-color(surface--complement--text)};
-  --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--complement--link)};
-  --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--complement--link--visited)};
-  --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--complement--link--hover)};
-  --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--complement--link--focus)};
-}
-
-:host([color="accent"]) {
-  --pfe-card--bg:                               #{pfe-color(surface--accent)};
-  --pfe-broadcasted--color--text:               #{pfe-color(surface--accent--text)};
-  --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--accent--link)};
-  --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--accent--link--visited)};
-  --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--accent--link--hover)};
-  --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--accent--link--focus)};
-}
+  // Broadcast color variables
+  --pfe-broadcasted--color--text:               #{pfe-color(surface--base--text)};
+  --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--base--link)};
+  --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--base--link--visited)};
+  --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--base--link--hover)};
+  --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--base--link--focus)};
 
-:host([size="small"]) {
-  --pfe-card--padding:        #{pfe-var(container-spacer)};
-}
+  //-- Deprecated but still supported
+  --pfe-card_header--size:  #{pfe-var(font-size--heading--gamma)};
 
-.pfe-card__header,
-.pfe-card__body,
-.pfe-card__footer {
-  display: block;
-  margin: 0;
-}
+  // Add backward support for previous variable naming
+  // @TODO deprecate padding and bg in favor of BEM naming
+  --pfe-card--padding:      #{pfe-local(Padding)};
+  --pfe-card--bg:           #{pfe-local(BackgroundColor)};
 
-.pfe-card__header::slotted(h1:first-child),
-.pfe-card__header::slotted(h2:first-child),
-.pfe-card__header::slotted(h3:first-child),
-.pfe-card__header::slotted(h4:first-child),
-.pfe-card__header::slotted(h5:first-child),
-.pfe-card__header::slotted(h6:first-child) {
-  margin-top: 0 !important; // Specificity issue. Why?!
-  font-size: var(--pfe-card_header--size);
-}
+  // Start of style declarations for host element
+  display: flex;
+  flex-direction: column;
+  justify-items: flex-start;
 
-.pfe-card__body::slotted(*:nth-child(2)) { // child pseudo-selectors appear to be based on slotted code before upgrade.
-  margin-top: 0 !important;
-}
+  padding:       #{pfe-local(Padding)};
+  border:        #{pfe-local(Border)};
```

Comment:
I don't know that we want a border around the card by default... can the default color be transparent? It creates a rim around the card header:
<img width="396" alt="screen shot 2019-02-19 at 4 43 12 pm" src="https://user-images.githubusercontent.com/456863/53049595-7ae9e180-3465-11e9-8c3b-4280216c9d7c.png">

Reply:
The border color is set to transparent already but the fact that it has width is pushing the header in.  I think I need to set the weight to 0px instead to fix this.

Reply:
Fixed this.